app/src/pages/inside/profilePage/profilePage.jsx

```diff
@@ -117,9 +117,11 @@ export class ProfilePage extends Component {
               onChangeTab={this.props.dispatch}
             />
           </div>
-          <div className={cx('footer')}>
-            <LocalizationBlock />
-            <DeleteAccountBlock />
+          <div>
```

Comment:
Why do we need this extra div wrapper here?

Reply:
Done.